packages/salesforcedx-vscode-replay-debugger/test/integration/config/data/apexTest.log

```diff
@@ -0,0 +1,481 @@
+41.0 APEX_CODE,FINEST;APEX_PROFILING,FINEST;CALLOUT,FINEST;DB,FINEST;SYSTEM,FINE;VALIDATION,INFO;VISUALFORCE,FINER;WAVE,FINEST;WORKFLOW,FINER
```

Comment:
@DatGuyJonathan - How did you generate this file? Is this really the trace from a single transaction? Or did you copy-and-paste several examples together?

Reply:
This is a real debug log from running a test class.